src/app/about/about.scss

```diff
@@ -1,13 +1,77 @@
-body {
-  .container {
+@use '../globals.scss' as *;
+
+.about-page {
+  padding-top: 6rem;
+  padding-bottom: 6rem;
+
+  &__heading-container {
     display: flex;
     flex-direction: column;
-    height: 100%;
-    margin-top: 2%;
+    align-items: center;
+    justify-content: center;
+    gap: 2rem;
+    width: 100%;
+  }
 
-    .about-header {
-      font-family: inherit;
-      font-weight: 700;
+  &__text {
+    font-size: 24px;
+    text-align: center;
+    letter-spacing: -0.48px;
+    color: #424242;
+    font-family: inherit;
+
+    @include tablet {
+      font-size: 1.5rem;
     }
   }
+
+  &__card-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 40px;
+    margin-top: 60px;
+    flex-wrap: wrap;
```

Comment:
```suggestion
    flex-wrap: wrap;
    
    @include mobile {
      gap: 0px;
    }
```

Reducing the gap on mobile when it's a single column

### Before

![image](https://github.com/CodeForBc/codeforbc-web/assets/16601729/bf311c87-3e3c-4e10-9149-2cd9e1fe4a98)

### After

![image](https://github.com/CodeForBc/codeforbc-web/assets/16601729/d8bd87c0-87bc-4504-9fc9-ea58b4decb06)